pkg/oc/cli/admin/release/image_mapper.go

```diff
@@ -175,7 +177,54 @@ func NewImageMapperFromImageStreamFile(path string, input *imageapi.ImageStream,
 		}
 		references[tag.Name] = ref
 	}
-	return NewImageMapper(references)
+	imageMapper, err := NewImageMapper(references)
+	if err != nil {
+		return nil, err
+	}
+
+	// load all version values from the input stream, including any defaults, to perform
+	// version substitution in the returned manifests.
+	versions := make(map[string]string)
+	tagsByName := make(map[string][]string)
+	for _, tag := range input.Spec.Tags {
+		if _, ok := references[tag.Name]; !ok {
+			continue
+		}
+		value := tag.Annotations["io.openshift.build.versions"]
+		if len(versions) > 0 {
```

Comment:
why call it versions when we exit after first version... ?

Reply:
Also if we drop the "io.openshift.build.versions" from the latest tag, how do you stop this from accepting version from older tags...?

Reply:
also we could break after we successfully set first versions from tags at the end [here](https://github.com/openshift/origin/pull/21889/commits/3ffdd932865239762d152c8c12d64ce3ea283b52#diff-5cb643d1d8350456bdef5ee588111ad3R211) right?

Reply:
I'm removing this check and adding an error condition when anyone redefines a version.  For now, two images may not declare different versions for the same name.  When we hit a use case that needs it, we'll potentially allow you to be scoped to your operator or get both.